github/resource_organization.go

```diff
@@ -67,3 +69,12 @@ func organizationFromAPI(apiObj *github.Organization) gitprovider.OrganizationIn
 		Description: apiObj.Description,
 	}
 }
+
+// validateOrganizationAPI validates the apiObj received from the server, to make sure that it is
+// valid for our use.
+func validateOrganizationAPI(apiObj *github.Organization) error {
+	if apiObj.Login == nil {
```

Comment:
This could probably use `validateAPIObject` like `validateDeployKeyAPI` does, wdyt?